app/src/main/java/com/babylon/wallet/android/presentation/onboarding/restore/backup/RestoreFromBackupScreen.kt

```diff
@@ -275,24 +280,22 @@ private fun RestoreFromBackupContent(
         }
     }
 
-    if (state.isPasswordSheetVisible) {
```

Comment:
I tested with 2 different phones and the contextual menu for pasting doesn't appear on either. Can it be related to the fact that the `TextField` has the `inputType` set to `password`? maybe then there is no reason to change the sheet implementation?

Reply:
I fixed the nav bars padding but as you can see in the video I can see the contextual menu.


https://github.com/user-attachments/assets/356b86f8-48dd-406e-a79a-7068bd236277

Reply:
just tested too and it works

Reply:
It seems that it's reproducible only on one of my devices (Pixel 6 with Android 14). Noted to test more and try to fix it when I get some spare time.